addons/hr_timesheet/views/project_portal_templates.xml

```diff
@@ -44,7 +44,7 @@
     </template>
 
     <template id="portal_tasks_list_inherit" inherit_id="project.portal_tasks_list" name="Portal: My Tasks with Timesheets">
-        <xpath expr="//t[@t-call='portal.portal_table']" position="inside">
+        <xpath expr="//t[@t-foreach='tasks']" position="inside">
```

Comment:
In fact, why not backport this fix? https://github.com/odoo/odoo/pull/100703/commits/094d6cf0cacd06fc85c59e94b80cd7988eae7f26

Reply:
I agree

Reply:
As both of them are producible in version 15.0 so fixed them in 15.0

Reply:
Okay but you did not do the changes 😅

Reply:
Done Thanks :+1: